### PR TITLE
Update to allow new naming conventions for electron

### PR DESCRIPTION
**Pull request by jessb321**
electron/electron#6838

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "license": "MIT",
   "dependencies": {
     "electron-zip-packager": "file:..",
+    "electron": "^1.3.1",
     "electron-prebuilt": "^1.3.1"
```
Comment (azu) [inline example/package.json]: Can we remove "electron-prebuilt" ?

**Comment by jessb321** [comment]
Yeah, sorry about that, Not sure how it slipped in. ¯\_(ツ)_/¯

**Comment by azu** [comment]
@willyb321 Good! Thanks.

**Comment by jessb321** [comment]
@azu No problem! Glad to help. (plus I wasn't able to update my own application.) ;)

**Comment by jessb321** [comment]
@azu btw don't you need to publish a new version? Thanks!

**Comment by azu** [comment]
Release https://github.com/azu/electron-zip-packager/releases/tag/3.0.0
Because, https://github.com/electron-userland/electron-packager/releases/tag/v7.6.0 contains a breaking change...

**Pull request merged**